modules/release-notes/pages/index.adoc

```diff
@@ -122,6 +122,28 @@ vertex function can also be used with vertex input parameters in the query-body
 
 == Fixed issues
 
+=== Fixed and Improved in 3.9.2-1
+
+==== Crashes and Deadlocks
+* Fixed GPE crash caused by aborted query still being used by running thread.
+* Fixed crash issue after RESTPP auth refresh.
+* Fixed GSE crash due to check failure when refreshing leader info after 60 minute timeout.
+* Fixed GPE crash and non-recovery due to Kafka and GPE offset comparison misread after GPE delta topic on related nodes were deleted.
+
+==== Functionality
+* Fixed data inconsistency when GSE consumed deletion requests from Kafka on update of `committed_next_id_map`
+via write queries when running `data_consistency_check` with GPE and GSE.
```

Comment:
```suggestion

```

I suggest we delete this line @victorleeTG . What do you think? I think it is redundant.